CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Version 1.0.1 - December 9, 2019
+================================
+- Update iOS SDK to 12.1.2
```

Comment:
Do we want to add notes about why this is a recommended update?

Reply:
We usually do not do that for framework updates.